pro/src/components/ReimbursementBankAccount/__specs__/ReimbursementBankAccount.spec.tsx

```diff
@@ -0,0 +1,198 @@
+import { render, screen } from '@testing-library/react'
+import React from 'react'
+
+import {
+  BankAccountResponseModel,
+  BankAccountApplicationStatus,
+} from 'apiClient/v1'
+
+import ReimbursementBankAccount from '../ReimbursementBankAccount'
+
+describe('ReimbursementBankAccount', () => {
+  let bankAccount: BankAccountResponseModel
+
+  beforeEach(() => {
+    bankAccount = {
+      id: 1,
+      isActive: true,
+      label: 'Bank account label',
+      obfuscatedIban: 'XXXX XXXX XXXX 0637',
+      bic: 'BDFEFRPP',
+      dsApplicationId: 6,
+      status: BankAccountApplicationStatus.ACCEPTE,
+      dateCreated: '2023-09-22T12:44:05.410448',
+      dateLastStatusUpdate: null,
+      linkedVenues: [{ id: 315, commonName: 'Le Petit Rintintin' }],
+    }
+  })
+
+  it('should render with pending bank account', async () => {
+    bankAccount.isActive = false
+    render(
+      <ReimbursementBankAccount
+        bankAccount={bankAccount}
+        venuesNotLinkedLength={0}
+        bankAccountsNumber={1}
+      />
+    )
+
+    expect(screen.getByText('Bank account label')).toBeInTheDocument()
+    expect(screen.getByText('IBAN : **** 0637')).toBeInTheDocument()
+    expect(screen.getByText('BIC : BDFEFRPP')).toBeInTheDocument()
+
+    expect(
+      screen.getByText(
+        'Compte bancaire en cours de validation par nos services'
+      )
+    ).toBeInTheDocument()
+    expect(
+      screen.queryByText('Lieu(x) rattaché(s) à ce compte bancaire')
+    ).not.toBeInTheDocument()
+  })
+
+  describe('Active bank account', () => {
+    beforeEach(() => {
+      bankAccount = {
+        id: 1,
+        isActive: true,
+        label: 'Bank account label',
+        obfuscatedIban: 'XXXX XXXX XXXX 0637',
+        bic: 'BDFEFRPP',
+        dsApplicationId: 6,
+        status: BankAccountApplicationStatus.ACCEPTE,
+        dateCreated: '2023-09-22T12:44:05.410448',
+        dateLastStatusUpdate: null,
+        linkedVenues: [{ id: 11, commonName: 'Le Petit Rintintin' }],
+      }
+    })
```

Comment:
tu peux enlever ce describe & le beforEach
je pense que le "pending bank acount" du it() précédent est suffisant